unime/src/routes/welcome/password/confirm/+page.svelte

```diff
@@ -12,12 +14,23 @@
   import Smiley from '~icons/ph/smiley';
   import SmileySad from '~icons/ph/smiley-sad';
 
-  let passwordsEqual: boolean | undefined;
+  // 3 states: true (match), false (mismatch), undefined (I don't know).
+  let passwords_match: boolean | undefined = undefined;
```

Comment:
Let's stick with the official naming convention where possible (camelCase). We're only using snake_case for types exported by Rust, but we should clean that up eventually too.

Reply:
Was not aware of this convention.